sim/icarus/add-arm64.patch

```diff
@@ -0,0 +1,3118 @@
+diff --git a/config.sub b/config.sub
```

Comment:
should we copy that from the autoconf package like you did for #303 ?

Reply:
Good idea. I've copied it over.